econml/inference.py

```diff
@@ -279,3 +429,521 @@ def prefit(self, estimator, *args, **kwargs):
         super().prefit(estimator, *args, **kwargs)
         # need to set the fit args before the estimator is fit
         self.model_final.cov_type = self.cov_type
+
+
+class InferenceResults(object):
+    """
+    Results class for inferences.
+
+    Parameters
+    ----------
+    d_t: int
+        Number of treatments
+    d_y: int
+        Number of outputs
+    pred : array-like, shape (m, d_y, d_t) or (m, d_y)
+        The prediction of the metric for each sample X[i].
+        Note that when Y or T is a vector rather than a 2-dimensional array,
+        the corresponding singleton dimensions should be collapsed
+        (e.g. if both are vectors, then the input of this argument will also be a vector)
+    pred_stderr : array-like, shape (m, d_y, d_t) or (m, d_y)
+        The prediction standard error of the metric for each sample X[i].
+        Note that when Y or T is a vector rather than a 2-dimensional array,
+        the corresponding singleton dimensions should be collapsed
+        (e.g. if both are vectors, then the input of this argument will also be a vector)
+    inf_type: string
+        The type of inference result.
+        It could be either 'effect', 'coefficient' or 'intercept'.
+    pred_dist : array-like, shape (b, m, d_y, d_t) or (b, m, d_y)
```

Comment:
How is b determined?

Reply:
And is this argument actually used anywhere?

Reply:
it will be used for bootstrap, not being used anywhere now.